model/metrics/faas-metrics.yaml

```diff
@@ -4,7 +4,7 @@ groups:
     metric_name: faas.invoke_duration
     brief: "Measures the duration of the function's logic execution"
     instrument: histogram
-    unit: "ms"
+    unit: "s"
```

Comment:
Nit: Wouldn't it be better if the text "this metric SHOULD be specified.." was a `note` here? I think those should be rendered by the tool.

Reply:
Could be changed. I went for consistency with HTTP metrics, where it's not a note: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md#metric-httpserverrequestduration